documentation/reference/qfieldcloud/jobs.md

```diff
@@ -0,0 +1,115 @@
+
+#### Triggers
+
+This job is triggered every time the **Download** or **Synchronize** buttons are pressed on QField. Unless at least one of the following condition are valid:
+
+- The project never run a `process_projectfile` job that resulted in `SUCCESS` status.
+- There is already a `package` job in `PENDING` status.
+- The project contains does not contain online vector layers (PostGIS, WFS etc), the latest `package` job result was `SUCCESS` and there were not file uploads, nor change uploads.
+
+#### Troubleshoot
+
+A `package` job might result in `FAILED` status. Check the non-exhaustive list of causes bellow:
+
+- The project never run a `process_projectfile` job that resulted in `SUCCESS` status.
+- Some of the project layers are inaccessible from QFieldCloud. Make sure all files are uploaded and all credentials to online layers (PostGIS, WFS etc) are stored within the QGIS project file.
+
+### Delta apply (`delta_apply`) job
+
+Delta apply jobs is responsible to make all pushed QField changes permanent.
+
+#### Triggers
+
+This job is triggered every time a **Synchronize** or **Push changes** buttons are pressed on QField, or **Apply pending changes** button is pressed on the **Changes** project page. If any of the following condition are valid:
+
+- The project never run a `process_projectfile` job that resulted in `SUCCESS` status.
+- There is already a `delta_apply` job in `PENDING` status.
+
+#### Troubleshoot
+
+A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive list of causes bellow:
+
+- At least one of the online databases (PostGIS/WFS) used in the QGIS project reset the connection.
+- The project is too big and the job has failed to run.
+- There are hidden files and directories within the project that are preventing the normal work of QFieldCloud. Hidden files and directories are those starting with a leading dot (`.`).
+
+
+## Troubleshoot job logs
+
+When running a job, usually you can find a step called "Check project layers" that prints a table with all the project layers and status next to them.
+
+The possible statuses are:
+
+- **ok** - The layer loads correctly on QFieldCloud.
+- **invalid_dataprovider** - the layer's data provider is invalid. Usually additional information is shown in the "Provider Summary".
+- **invalid_layer** - This errors should happen very rarely if ever. The data is loaded correctly, but for some reason QGIS reports the layer as invalid.
+
+
+### Unable to connect to service "`{SERVICE}`".
+
+QFieldCloud tries to connect to a PostgreSQL service that is not available. You should need to create a new pgservice [secret](projects.md#secrets) so QFieldCloud can connect to the PostGIS service.
+
+### Unable to connect to host "`{HOST}`".
+
+QFieldCloud cannot establish a connection to the given `{HOST}`. Your service is not accessible from the QFieldCloud server. You might been to ask your IT department to whitelist the QFieldCloud IP.
```

Comment:
```suggestion
QFieldCloud cannot establish a connection to the given `{HOST}`. Your service is not accessible from the QFieldCloud server. You might need to ask your IT department to whitelist the QFieldCloud IP.
```

Are we able to link to the qfc ip here?

Reply:
Good idea, however I prefer to not put it in the docs until we migrate to k8s and we fix the IP.

Reply:
What do you think about a skeleton page that shows the state of affairs that can be adjusted later?

Reply:
If you mean an new page in the docs, then yes, I do agree. However, I believe it would be out of scope of the current PR.

Reply:
A new page with technical specs would be great. Even if minimal with a "soon to be announced".

Reply:
See https://github.com/opengisch/QField-docs/pull/200